devicon.json

```diff
@@ -925,6 +925,25 @@
         "color": "#DD732A",
         "aliases": []
     },
+    {
+        "name": "flutter",
+        "tags": [
+            "framework",
+            "sdk"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```

Comment:
I'm missing the `plain-wordmark` version which you specified here. Have you forgotten to commit it? :) That's the reason why the bot:peek generation fails